### ISSUE TITLE
Load File in REPL throws exception

### ISSUE THREAD

**Issue by MrGung** [cursive-ide/cursive#2664]
Using 1.12.2-eap6-2021.3 and babashka 0.7.6, getting

```
java.lang.NullPointerException
	at cursive.scope$contains_QMARK_.invokeStatic(scope.clj:47)
	at cursive.scope$contains_QMARK_.invoke(scope.clj:46)
	at cursive.repl.actions$validate_load_file.invokeStatic(actions.clj:436)
	at cursive.repl.actions$validate_load_file.invoke(actions.clj:431)
	at cursive.repl.actions$load_file.invokeStatic(actions.clj:445)
	at cursive.repl.actions$load_file.invoke(actions.clj:441)
	at clojure.lang.Var.invoke(Var.java:384)
	at cursive.api.DelayedFn.invoke(DelayedFn.java:31)
	at cursive.actions.ClojureActionBase.actionPerformed(ClojureActionBase.java:38)
	at com.intellij.openapi.keymap.impl.ActionProcessor.performAction(ActionProcessor.java:65)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher$1.performAction(IdeKeyEventDispatcher.java:573)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.lambda$doPerformActionInner$10(IdeKeyEventDispatcher.java:706)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareWithCallbacks(ActionUtil.java:265)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.doPerformActionInner(IdeKeyEventDispatcher.java:702)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.processAction(IdeKeyEventDispatcher.java:645)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.processAction(IdeKeyEventDispatcher.java:584)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.processActionOrWaitSecondStroke(IdeKeyEventDispatcher.java:467)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.inInitState(IdeKeyEventDispatcher.java:456)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.dispatchKeyEvent(IdeKeyEventDispatcher.java:224)
	at com.intellij.ide.IdeEventQueue.dispatchKeyEvent(IdeEventQueue.java:804)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:754)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$6(IdeEventQueue.java:447)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:818)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$7(IdeEventQueue.java:446)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:805)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:498)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```

on Load File in REPL.

**Comment by cursive-ide** [comment]
I believe this is the same root cause as #2927.

**Comment by cursive-ide** [comment]
Fix released in 1.13.4-eap8 on 2024-07-23.

**Comment by cursive-ide** [comment]
Fix released in 1.13.4 on 2024-08-06.
